query_parameters.go

```diff
@@ -0,0 +1,60 @@
+// Licensed to ClickHouse, Inc. under one or more contributor
```

Comment:
We don't think we need this ?

Reply:
This is cherry picked from click house driver to support protocol 54459. since we bump the client protocol to 54461 for uuid.